### DISCUSSION TITLE
A Vine in Time Saves Nine! A Tuxemon Story

### DISCUSSION THREAD

**Discussion by Sanglorian** [Tuxemon/Tuxemon#2324] [General]
Hi folks, 

I wrote a short story for young adults set in the Tuxemon world. I've posted it [on the wiki](https://wiki.tuxemon.org/A_Vine_in_Time_Saves_Nine) in case anyone's interested.

**Comment by Davidliang11292** [comment]
Great story! I liked it very much, and what surprised me more was that Linus' first Tuxemon was not a Rockitten!

**Reply by Sanglorian** [reply]
Thanks :D I'm glad you liked it

**Comment by HippasusTwo** [comment]
Nice. I was also surprised.
